src/components/itemMediaInfo/itemMediaInfo.js

```diff
@@ -113,7 +113,7 @@ const attributeDelimiterHtml = layoutManager.tv ? '' : '<span class="hide">: </s
             if (stream.Profile) {
                 attributes.push(createAttribute(globalize.translate('MediaInfoProfile'), stream.Profile));
             }
-            if (stream.Level) {
+            if (stream.Level && stream.Type === 'Video') {
```

Comment:
Do we have to check `Type`?
Most attributes don't have such a check.

Reply:
Yes, only for certain attributes. Such as level should only be shown for video type but not for music and image.

Reply:
Do we have `Level`, `AverageFrameRate` and `RealFrameRate` for non-video and `BitRate` for non-video/audio?

Reply:
Nope. The first three are video only and bitrate is audio/video only.

Reply:
Actually, I got `Level` for audio, but it is `0`.
I mean, maybe remove these checks?

Reply:
AKAIK 0 and -99 are usually invalid levels, compared with 51(5.1 h264) or 180(6.0 hevc). Did we ever use level attribute for audio?
https://github.com/jellyfin/jellyfin/blob/50bc41d84d7f3f7ecdccbb6ed1885426f87c0d9a/tests/Jellyfin.MediaEncoding.Tests/Test%20Data/Probing/music_metadata.json

Reply:
I have video VP9 with `Level: -99`.
Subtitles and audio have `Level: 0`.
I think we can just leave `if (Level)` or check for validity `if (Level > 0)`.

Reply:
Done.